web/filament-js/jsbindings.cpp

```diff
@@ -286,6 +286,7 @@ class_<Engine>("Engine")
     .function("getTransformManager", EMBIND_LAMBDA(TransformManager*, (Engine* engine), {
         return &engine->getTransformManager();
     }), allow_raw_pointers())
+
```

Comment:
I only use vertical whitespace in order to group create / destroy pairs.

Note that all destroy methods do not necessarily have corollary create methods.